pkg/actors/internal/placement.go

```diff
@@ -83,7 +83,7 @@ type ActorPlacement struct {
 }
 
 func addDNSResolverPrefix(addr []string) []string {
-	resolvers := []string{}
+	resolvers := make([]string, len(addr))
```

Comment:
```suggestion
	resolvers := make([]string, 0, len(addr))
```
I made a mistake in my previous recommendation since that will make resolves having len(addr) from the beginning.